specification/network/resource-manager/readme.md

```diff
@@ -31,24 +31,77 @@ openapi-type: arm
 tag: package-preview-2022-06
```

Comment:
```suggestion
tag: package-2022-06-preview
```

Reply:
According to the discussion with network team, this preview version is used by our own team only. So we won't change the default version tag.

Reply:
Then why are you changing the existing default tag ? If you only want a preview tag, please add a new one and do not change the default tag.

Reply:
It doesn't allow to set a default tag to un-existed tag. Either update default tag value correctly or keep original default tag definition.

Reply:
I didn't make this change, and I guess it was modified when auto generating the preview version branch. Fixed it now, Thanks